tests/test_cube_equaliser.py

```diff
@@ -1,133 +1,192 @@
+        cube_1 = base_cube[0:2]
+        cube_2 = base_cube[2:4]
+        cube_3 = base_cube[4:]
+        height_coord = iris.coords.AuxCoord(2,
+                                            standard_name='height',
+                                            long_name='height',
+                                            var_name='height',
+                                            units='m')
+        cube_2.add_aux_coord(height_coord)
+        self.temp_1_aux = 'temp_1_aux.nc'
+        self.temp_2_aux = 'temp_2_aux.nc'
+        self.temp_3_aux = 'temp_3_aux.nc'
+        iris.save(cube_1, self.tmp_dir_aux + self.temp_1_aux)
+        iris.save(cube_2, self.tmp_dir_aux + self.temp_2_aux)
+        iris.save(cube_3, self.tmp_dir_aux + self.temp_3_aux)
+
+    def test_equalise_attributes(self):
+        glob_path = self.tmp_dir_attr + '*.nc'
+        print(glob_path)
+        filepaths = glob(glob_path)
+        test_load = [iris.load_cube(cube) for cube in filepaths]
+        test_load = ch.equalise_attributes(test_load)
+        for cubes in test_load:
+            self.assertEqual(cubes.attributes, test_load[0].attributes)
+
+    def test_equalise_time_units(self):
+        glob_path = self.tmp_dir + '*.nc'
+        print(glob_path)
+        filepaths = glob(glob_path)
+        test_load = [iris.load_cube(cube) for cube in filepaths]
+        test_load = ch.equalise_time_units(test_load)
+        for index, cube in enumerate(test_load):
+            for time_coords in cube.coords():
+                if time_coords.units.is_time_reference():
+                    self.assertEqual(cube[index].units.calendar,
+                                     cube[index-1].units.calendar)
+
+    def test_remove_attributes(self):
+        glob_path = self.tmp_dir + '*.nc'
+        print(glob_path)
+        filepaths = glob(glob_path)
+        test_load = [iris.load_cube(cube) for cube in filepaths]
+        ch.remove_attributes(test_load)
+        keys = list(test_load[0].attributes.keys())
+        for cube in test_load:
+            for key in keys:
+                self.assertEqual(cube.attributes[key], '')
+
+    def test_equalise_data_type(self):
+        glob_path = self.tmp_dir + '*.nc'
+        filepaths = glob(glob_path)
+        test_load = [iris.load_cube(cube) for cube in filepaths]
+        ch.equalise_data_type(test_load)
+        for cube in test_load:
+            self.assertEqual(cube.dtype, 'float32')
+        ch.equalise_data_type(test_load, 'float64')
+        for cube in test_load:
+            self.assertEqual(cube.dtype, 'float64')
+        ch.equalise_data_type(test_load, 'int32')
+        for cube in test_load:
+            self.assertEqual(cube.dtype, 'int32')
+        ch.equalise_data_type(test_load, 'int64')
+        for cube in test_load:
+            self.assertEqual(cube.dtype, 'int64')
+
+    def test_equalise_dim_coords(self):
+        glob_path = self.tmp_dir + '*.nc'
```

Comment:
Is there a difference in the dim_coords names when they're first loaded? Could you edit one cube to check that this is definitely making a change please?